package.json

```diff
@@ -49,6 +49,11 @@
       "packages/*"
     ]
   },
+  "pnpm": {
+    "overrides": {
+      "@rspack/core": "0.4.0"
```

Comment:
Maybe we should downgrade Rsbuild version? Pnpm overrides can not work for Rspress users.

Reply:
That means we should revert the following pr:
- https://github.com/web-infra-dev/rspress/pull/440
- https://github.com/web-infra-dev/rspress/pull/439

Reply:
yeah...It may be more better to push Rspack to fix the performance issue